### PR TITLE
chart: support Talos Linux

### PR DESCRIPTION
**Pull request by kvaps**
# Pull Request

- [ ] Make sure you have followed [Kube-OVN Code Style](https://github.com/kubeovn/kube-ovn/blob/master/CODE_STYLE.md).

## What type of this PR

Examples of user facing changes:

- Features
- Bug fixes
- Docs
- Tests

Add support for installing Kube-OVN on Talos Linux
It also requires https://github.com/kubeovn/kube-ovn/pull/3585 to work

## Which issue(s) this PR fixes

Fixes #(issue-number)

## WHAT

copilot:summary

copilot:poem

## HOW

copilot:walkthrough

### CODE REVIEW

File: charts/templates/ovsovn-ds.yaml [hunk 1]
```diff
@@ -45,8 +45,18 @@ spec:
           command: ["/kube-ovn/start-ovs-dpdk.sh"]
           {{- else }}
           command: 
+          {{- if .Values.DISABLE_MODULES_MANAGEMENT }}
+          - /bin/sh
+          - -c
```
Comment (zhangzujian) [inline charts/templates/ovsovn-ds.yaml:50]:
```suggestion
          - -ec
```

**Comment by kvaps** [comment]
@zhangzujian thanks, PR rebased

**Review by zhangzujian — APPROVED** [review]

**Comment by zhangzujian** [comment]
Please resolve the conflict.

**Comment by kvaps** [comment]
> Please resolve the conflict.
Sure, done

**Pull request merged**